architecture/additional_concepts/flannel.adoc

```diff
@@ -29,4 +31,9 @@ responsibile for:
 
 Each *flanneld* agent provides this infomation to a centralized *etcd* store so
 other agents on hosts can create an overlay network and route packets to
-any container within the *flannel* network.
\ No newline at end of file
+any container within the *flannel* network.
+
+The following diagram illustrates the architecture and data flow from one
+container to another using a *flannel* network:
+
+image::flannel.png[Flannel Communication]
```

Comment:
@danwinship -- from what I've seen, host-gw sets up a series of routes to direct packets, which is why I had the UDP encapsulation to be removed from the original image here:

https://raw.githubusercontent.com/coreos/flannel/master/packet-01.png

So maybe the image is still usable but in the text we need to specify that the default ansible install uses host-gw mode, which requires direct L2 connection between all nodes so that the packets get routed accordingly. What do you think?